### PR TITLE
CI - Allow highest PHPUnit version for each PHP version

### PR DESCRIPTION
**Pull request by mondrake**

### CODE REVIEW

File: tests/AbstractTestCase.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+namespace UAParser\Test;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractTestCase extends TestCase
+{
+    /**
+     * Polyfill to allow exception testing across PHPUnit 4 to PHPUnit 7.
+     */
+    public function fcExpectException($exception, $message = null)
```
Comment (lstrojny) [inline tests/AbstractTestCase.php]: Can we just polyfill setExpectedException()?

Comment (mondrake) [reply tests/AbstractTestCase.php]: Maybe with a trait. I will check.

Comment (mondrake) [reply tests/AbstractTestCase.php]: Ah no, traits are available as of PHP 5.4. Uaparser tests on 5.3 also.

Comment (mondrake) [reply tests/AbstractTestCase.php]: So on 2nd read we do not need a trait, just provide the method in a base class.

Comment (lstrojny) [reply tests/AbstractTestCase.php]: Yup, that should work

**Comment by mondrake** [comment]
It’s done

File: tests/AbstractTestCase.php (hunk 1 above)
Comment (mimmi20) [reply tests/AbstractTestCase.php]: The Namespaced version for PHPUnit is available in the versions `^4.8.35 | ^5.4.3 | ^6.0.0 | ^7.0.0`.

**Comment by lstrojny** [comment]
Thank you!

**Pull request merged**